staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -61,6 +73,21 @@ type CustomResourceDefinitionVersion struct {
 	// Storage flags the version as storage version. There must be exactly one flagged
 	// as storage version.
 	Storage bool
+	// Schema describes the schema for CustomResource used in validation, pruning, and defaulting.
+	// Top-level and per-version schemas are mutually exclusive.
+	// Per-version schemas may not all be set to identical values (top-level validation schema should be used instead)
+	// +optional
```

Comment:
for all the alpha, feature-gated fields, add both of those details to the documentation. For example:

```
// This field is alpha-level and is only honored by servers that enable the XYZ feature.
```